tracer/src/Datadog.Trace/Iast/StackWalker.cs

```diff
@@ -83,34 +85,34 @@ public static bool AssemblyExcluded(string assembly)
         ExcludedAssemblyCache[assembly] = excluded;
 
         return excluded;
-    }
 
-    // For performance reasons, we are not supporting wildcards fully. We just need to use '.' at the end for now. We can use regular expressions
-    // if in the future we need a more sophisticated wildcard support
-    private static bool IsExcluded(string assembly)
-    {
-        foreach (var assemblyToSkip in AssemblyNamesToSkip)
+        // For performance reasons, we are not supporting wildcards fully. We just need to use '.' at the end for now. We can use regular expressions
+        // if in the future we need a more sophisticated wildcard support
+        static bool IsExcluded(string assembly)
         {
+            foreach (var assemblyToSkip in AssemblyNamesToSkip)
+            {
 #if NETCOREAPP3_1_OR_GREATER
             if (assemblyToSkip.EndsWith('.'))
```

Comment:
nit: this needs indenting too
```suggestion
                if (assemblyToSkip.EndsWith('.'))
```